context/context.js

```diff
@@ -13,20 +13,43 @@ export const CMProvider = ({children}) => {
       console.log(error.message);
     }
   };
-  const getNews = async () => {
-    try {
-      const res = await fetch("/api/bind-api");
-      const data = await res.json();
 
-      return data.data.value;
-    } catch (error) {
-      console.log(error.message);
-    }
-  };
+  return <CMContext.Provider value={{getCoins}}>{children}</CMContext.Provider>;
+};
+
+const ActivityContext = createContext();
+
+export function ActivityProvider({children}) {
```

Comment:
You are using React Context here and a 3rd party state management library (`react-hooks-global-state
`) for other parts of your state. Please choose only one state management technique and manage your global state with it.

Reply:
done!